packages/emails/templates/attendee-daily-video-download-recording-email.ts

```diff
@@ -43,16 +44,32 @@ export default class AttendeeDailyVideoDownloadRecordingEmail extends BaseEmail
     return this.attendee.timeZone;
   }
 
+  protected getLocale(): string {
+    return this.attendee.language.locale;
+  }
+
   protected getInviteeStart(format: string) {
-    return this.getRecipientTime(this.calEvent.startTime, format);
+    return this.getFormattedRecipientTime({
+      time: this.calEvent.startTime,
+      format,
+      locale: this.getLocale(),
```

Comment:
If locale and timezone are on the instance then can we avoid passing these and calculate in `getFormattedRecipientTime` itself?

Reply:
we can't calculate this in getFormattedRecipientTime because locale and timezone is different in both organiser and attendee . For eg. this.attendee.language.locale and this.calEvent.organizer.language.locale.